include/Contact.php

```diff
@@ -1,7 +1,5 @@
 <?php
 
-require_once('include/Probe.php');
```

Comment:
Why had you removed it from here? According to @fabrixxm we should do all the includes at the top of the files - while in the past we did the opposite.

I don't know which is better - but we should do this consistently.